src/mme-app/actionHandlers/defaultMmeProcedureActionHandlers.cpp

```diff
@@ -307,13 +308,11 @@ ActStatus ActionHandlers::default_ddn_handler(ControlBlock& cb)
 
     if (gtpCause != GTPV2C_CAUSE_REQUEST_ACCEPTED)
     {
-        SessionContext *sess_p = ueCtxt->getSessionContext();
         struct DDN_ACK_Q_msg ddnAck;
         ddnAck.msg_type = ddn_acknowledgement;
-        ddnAck.s11_sgw_cp_teid = sess_p->getS11SgwCtrlFteid().fteid_m.header.teid_gre;
+	/*Incase of unavailability of session/UE Contexts , s11_sgw_cp_teid will be set as 0 */
+        ddnAck.s11_sgw_cp_teid = sgw_cp_teid;
```

Comment:
We need to inform S11 on which ip to send the ddn ack on. Thats the reason we were sending this s11 fteid parameter. In case when Mme talks to 2 SPGWs, using the global spgw ip will create issues and sends messages to wrong spgw.

Reply:
Taken care in f72e5b1